cmd/shortener/main.go

```diff
@@ -3,38 +3,70 @@ package main
 import (
 	"log"
 	"net/http"
+	"os"
 
 	"github.com/IgorGreusunset/shortener/cmd/config"
 	model "github.com/IgorGreusunset/shortener/internal/app"
 	"github.com/IgorGreusunset/shortener/internal/handlers"
+	"github.com/IgorGreusunset/shortener/internal/logger"
+	"github.com/IgorGreusunset/shortener/internal/middleware"
 	"github.com/IgorGreusunset/shortener/internal/storage"
 	"github.com/go-chi/chi/v5"
 )
 
 
+func main() {
 
+	config.ParseFlag()
 
-func main() {
 	router := chi.NewRouter()
 
+	logger.Initialize()
+
+
+	//Открываем файл-хранилище
+	file, err := os.OpenFile(config.File, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	if err != nil {
+		log.Fatalf("Error during opening file with shorten urls: %v", err)
+	}
+
+	//Создаем новое хранилище
 	db := storage.NewStorage(map[string]model.URL{})
 
+	//Наполняем хранилище данными из файла
+	err = db.FillFromFile(file)
+	if err != nil {
+		logger.Log.Infof("Error during reading from file with shorten urls: %v", err)
+	}
+
+
+	file.Close()
```

Comment:
Возможно стоит вынести в отдельную функцию для инициализации из файла. В этом случае так же можно было бы использовать defer для закрытия файла.

```suggestion
	file, err := os.OpenFile(config.File, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
	defer file.Close()
	if err != nil {
		log.Fatalf("Error during opening file with shorten urls: %v", err)
	}

	//Создаем новое хранилище
	db := storage.NewStorage(map[string]model.URL{})

	//Наполняем хранилище данными из файла
	err = db.FillFromFile(file)
	if err != nil {
		logger.Log.Infof("Error during reading from file with shorten urls: %v", err)
	}
```